data/INRA/data5/end_of_alcoholic_fermentation.ttl

```diff
@@ -0,0 +1,7294 @@
+ 
+<data/winemaking/PechRouge/EndFermentationMonitoring/2016> a qb:DataSet; 
+  rdfs:label "Data about Fermentation done on Pech Rouge in 2016"; 
+  qb:structure <DSD/EndFermentationMonitoring/2016> . 
+ 
+<DSD/EndFermentationMonitoring/2016> a qb:DataStructureDefinition; 
+  qb:component [qb:dimension bdg:fermentationTank;        ]; 
```

Comment:
While `; ]` is syntactically correct (it indicates a missing prop-obj pair), it is useless and potentially confusing. Don't use such "empty semicolons"

Reply:
Fixed in 6bae82ef7f1df282ecf42c60bc602c2805a51a5a